demo/config/initializers/view_component_preview.rb

```diff
@@ -0,0 +1,10 @@
+module RenderWithLocals
```

Comment:
a little hack to retrieve and pass preview method arguments to `locals` in `render_with_template` method